src/Mono.Android/Java.Lang/Object.cs

```diff
@@ -91,21 +93,29 @@ internal void SetHandleOnDeserialized (StreamingContext context)
 		}
 
 #if JAVA_INTEROP
+		[DebuggerBrowsable (DebuggerBrowsableState.Never)]
+		[EditorBrowsable (EditorBrowsableState.Never)]
 		public int JniIdentityHashCode {
 			get {return (int) key_handle;}
 		}
 
+		[DebuggerBrowsable (DebuggerBrowsableState.Never)]
+		[EditorBrowsable (EditorBrowsableState.Never)]
 		public JniObjectReference PeerReference {
 			get {
 				return new JniObjectReference (handle, (JniObjectReferenceType) handle_type);
 			}
 		}
 
+		[DebuggerBrowsable (DebuggerBrowsableState.Never)]
+		[EditorBrowsable (EditorBrowsableState.Never)]
 		public virtual JniPeerMembers JniPeerMembers {
 			get { return _members; }
 		}
 #endif  // JAVA_INTEROP
 
+		[DebuggerBrowsable (DebuggerBrowsableState.Never)]
+		[EditorBrowsable (EditorBrowsableState.Never)]
 		public IntPtr Handle {
```

Comment:
Would this also hide usage of `Handle` like:
```csharp
private Java.Lang.Object foo;
//...
if (foo != null && foo.Handle != IntPtr.Zero) {
   //...
}
```
If so, maybe we shouldn't hide `Handle` as developers might often use it?

Xamarin.Forms uses `Handle` in these extension methods a lot: 

https://github.com/xamarin/Xamarin.Forms/blob/954cc8b0f174f179fd3395b4b1f257201f9701af/Xamarin.Forms.Platform.Android/Extensions/JavaObjectExtensions.cs#L7-L31

Reply:
Yes, it would hide `Handle` in that case (from Intellisense).  My gut says users wouldn't be using 'Handle' if they are writing applications or libraries.  (The authors of XF itself are a special case.)

But that's just an assumption and it's certainly up for discussion.

Reply:
You also might use `Handle` if you are working with JNI directly? Some folks might do that?